pkg/rkenodeconfigserver/nodeserver.go

```diff
@@ -584,3 +598,22 @@ func formatSystemImages(originSystemImages v3.WindowsSystemImages, windowsReleas
 
 	return shadow.Interface().(v3.WindowsSystemImages)
 }
+
+func toJSON(options interface{}) string {
+	result := "{}"
+	if options != nil {
+		bs, err := json.Marshal(options)
+		if err == nil {
+			return string(bs)
+		}
+	}
+	return result
+}
+
+func atoi(val string, defaultVal int) int {
+	ret, err := strconv.Atoi(val)
+	if err == nil {
+		return ret
+	}
+	return defaultVal
```

Comment:
This function swallow the error and return default value instead. We should return the error.

Reply:
This function is the same as [`func atoiWithDefault` commit for rke#1290](https://github.com/rancher/rke/pull/1290/files#diff-998130628a5d8a880484b379b1dcd47e). The `atoiWithDefault` of rke is for Linux node and the `atoi` in here is for Windows node. Let us name them same and always return the error, right?

Reply:
yes